sdks/urbackend-python/urbackend/db.py

```
@@ -0,0 +1,425 @@
+                return result["items"]
+            if isinstance(result, list):
+                return result
+            return []
+        except NotFoundError:
+            return []
+
+    def count(
+        self,
+        collection: str,
+        *,
+        filter: Optional[Dict[str, Any]] = None,
```

Ruff (0.15.11)

[error] Function argument `filter` is shadowing a Python builtin

(A002)